src/components/Crypto/styles.tsx

```diff
@@ -0,0 +1,81 @@
+import styled from 'styled-components';
+
+export const CryptoContainer = styled.View`
+  alignself: center;
+  height: 100px;
+  width: 85%;
+  marginbottom: 10px;
```

Comment:
what is this style? it works? 

shouldn't it be margin-bottom, font-size, etc... ?